### ISSUE TITLE
[DEPRECATION WARNING]: The comma separated role spec format

### ISSUE THREAD

**Issue by joestewart** [geerlingguy/ansible-role-elasticsearch#19]
Low priority.
```
- geerlingguy.elasticsearch was installed successfully
[DEPRECATION WARNING]: The comma separated role spec format, use the 
yaml/explicit format instead..
This feature will be removed in a future 
release. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
```

**Comment by leifmadsen** [comment]
I'm pretty sure this is the result of this open bug:  https://github.com/ansible/ansible/pull/14612

**Comment by geerlingguy** [comment]
Yeah, I'm going to have to wait on the upstream answer for how the roles should be listed under dependencies.

**Comment by geerlingguy** [comment]
Going to close this until something is done upstream. For now it should still work fine.
